patterns/2-structured/circle-communities.md

```diff
@@ -0,0 +1,81 @@
+* **Use Collaborative Tools**: Leverage collaborative tools and platforms (e.g., Jitsi Meet, Slack, GitHub) to facilitate communication, documentation and knowledge sharing.  Insights learned can be optionally shared to social channels such as Slack and LinkedIn, ensuring privacy.
+* **Foster a Culture of Continuous Improvement**: Regularly solicit feedback from members to identify areas for improvement. Use this feedback to refine the conditioning and activities of the Circle Community.
+
+## Resulting Context
+
+Embracing this pattern helps InnerSource programs become vibrant, effective and sustainable. Participants will feel more connected and engaged, leading to increased knowledge sharing, innovation and collective growth.
+
+## Related Patterns
+
+* [Dedicated Community Leader](https://patterns.innersourcecommons.org/p/dedicated-community-leader)
+* [Core Team](https://patterns.innersourcecommons.org/p/core-team)
+* [Maturity Model](https://patterns.innersourcecommons.org/p/maturity-model)
+* [Agile InnerSource Dojo](https://patterns.innersourcecommons.org/p/agile-innersource-dojo)
+
+## Known Instances
+
+* Dojo Circle
+  * Fridays at 13:00 UTC at <https://circle.dojo.center>
+  * Hosted by [Dojo Center](https://dojo.center)
+  * Engaged via [InnerSource Commons Foundation](https://innersourcecommons.org) in Slack at ```#dojo-center```
+  * Affiliated with the Mindset, Agile and InnerSource sub-communities within _InnerSource Dojo from SAP_
+* [InnerSource Dojo from SAP](https://www.youtube.com/watch?v=fXoVm5iTSCc)
+* _Unplugged InnerSource_ sessions within the _Reuse & InnerSource_ workgroup of [SAP Cross-Platform Architecture](https://community.sap.com/t5/technology-blogs-by-sap/cross-product-architecture-embracing-conway-s-law-for-better-software/ba-p/13648600)
+
+## Status
+
+* Initial
+
+## Authors
+
+* [Michael Basil](https://www.linkedin.com/in/michaelrbasil)
+* [Bill Westfall](https://www.linkedin.com/in/bill-westfall-3268494)
+* [Jim Manzzullo](https://www.linkedin.com/in/jimmanzzullo)
+* [Jeremy McMillan](https://www.linkedin.com/in/jeremymcm)
+* [AnuOlu Balogun](https://www.linkedin.com/in/anuolu)
+
+## Acknowledgements
+
+* [Guilherme Dellagustin](https://www.linkedin.com/in/dellagustin)
+* InnerSource Dojo from SAP:
+  * [Ryan Uttech](https://www.linkedin.com/in/ryanuttech)
+  * [Ignacio Pedragon](https://www.linkedin.com/in/ignaciopedregon)
+  * [Dale Sackrider](https://www.linkedin.com/in/sackrider)
+  * [Jason Fisher](https://www.linkedin.com/in/jasonrfisher)
+  * [Michael Ward](https://www.linkedin.com/in/michaelwardqa)
+  * [Alfred Darby III](https://www.linkedin.com/in/alfred-darby-iii-21b6683)
+* [Nick Basil](https://www.linkedin.com/in/nick-basil)
+* [Allison Kaufmann](https://www.linkedin.com/in/allisonkaufmann29)
+* [Dojo Consortium](https://dojoconsortium.org)
+* [Thingy Farm](https://thingyfarm.com/)
```

Comment:
```suggestion
* [Thingy Farm](https://thingyfarm.com/)

# Alias

* [Community of Practice](https://en.wikipedia.org/wiki/Community_of_practice)
* [Guilds](https://engineering.atspotify.com/2014/03/spotify-engineering-culture-part-1/)
```

IMO this is very similar to Communities of Practice, and potentially Guilds too.